.travis.yml

```diff
@@ -36,7 +36,13 @@ jobs:
         apt:
           packages:
             - libxml2-utils
-      script: xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd lib/Doctrine/ruleset.xml
+      script:
+        - |
+          if dpkg --compare-versions "$(composer show -n | grep squizlabs/php_codesniffer | tr -s ' ' | cut -d ' ' -f 2)" '>=' "3.3.2"; then
+            xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd lib/Doctrine/ruleset.xml
+          else
+            echo "Validation skipped, PHPCS 3.3.2+ required"
```

Comment:
Addresses this problem: https://github.com/doctrine/coding-standard/pull/84#issuecomment-419741589

Reply:
So what is the 3.3.2 change which the XML validation depends on?

Reply:
https://github.com/squizlabs/PHP_CodeSniffer/issues/2151

They forgot to update the schema to include the new `<element>` syntax for arrays (available since 3.3.0).

Reply:
Gotcha. It'd be nice if they had validation internally as in PHPUnit (https://github.com/sebastianbergmann/phpunit/issues/3048). Otherwise, the schema validity will have to be enforced by 3rd party's CI like ours.

Reply:
Possibly something for PHPCS 4.0. :)